app/src/test/java/com/jalasoft/routesapp/cities/CityPickerViewModelTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.jalasoft.routesapp.cities
+
+import com.jalasoft.routesapp.data.source.CountriesFakedata
+import com.jalasoft.routesapp.data.source.FakeDataCountryManager
+import com.jalasoft.routesapp.ui.cityPicker.viewModel.CityPickerViewModel
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import dagger.hilt.android.testing.HiltTestApplication
+import junit.framework.TestCase
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = HiltTestApplication::class, manifest = Config.NONE)
+@HiltAndroidTest
+class CityPickerViewModelTest : TestCase() {
+    @get :Rule
+    val hiltRule = HiltAndroidRule(this)
+
+    private lateinit var viewModel: CityPickerViewModel
+    private lateinit var fakeManager: FakeDataCountryManager
+
+    @Before
+    public override fun setUp() {
+        super.setUp()
+        hiltRule.inject()
+        fakeManager = FakeDataCountryManager()
+        viewModel = CityPickerViewModel(fakeManager)
+    }
+
+    @Test
+    fun `Fetch all the cities`() {
```

Comment:
Provide a description so we all know what the test is doing, like given... when... then...